symbolic-examples/stats/ByteString.And.32

```diff
@@ -1,4 +1,5 @@
 ByteString.And.32
-Number of constraints: 287
+Number of constraints: 254
 Number of variables: 286
-Number of range lookups: 0
\ No newline at end of file
+Number of all lookups: 0
```

Comment:
How is it possible that `and` as defined above doesn't introduce any range lookups?